src/lib/server/database/models/index.ts

```diff
@@ -51,13 +51,34 @@ export const confession = app.table(
         approvedAt: timestamp('approved_at', { withTimezone: true }).defaultNow(),
         authorId: bigint('author_id', { mode: 'bigint' }).notNull(),
         content: text('content').notNull(),
+        attachmentId: bigint('attachment_id', { mode: 'bigint' }).references(() => attachmentData.attachmentId),
     },
-    ({ confessionId, channelId }) => [uniqueIndex('confession_to_channel_unique_idx').on(confessionId, channelId)],
+    ({ confessionId, channelId, attachmentId }) => [
+        uniqueIndex('confession_to_channel_unique_idx').on(confessionId, channelId),
+        uniqueIndex('confession_to_attachment_unique_idx').on(confessionId, attachmentId),
+    ],
 );
 
 export type Confession = typeof confession.$inferSelect;
 export type NewConfession = typeof confession.$inferInsert;
 
 export const confessionRelations = relations(confession, ({ one }) => ({
     channel: one(channel, { fields: [confession.channelId], references: [channel.id] }),
+    attachment: one(attachmentData, { fields: [confession.attachmentId], references: [attachmentData.attachmentId] }),
 }));
+
+export const attachmentData = app.table('attachment_data', {
+    attachmentId: bigint('id', { mode: 'bigint' }).notNull().primaryKey(),
+    filename: text('filename').notNull(),
+    title: text('title'),
+    description: text('description'),
+    contentType: text('content_type'),
+    size: integer('size').notNull(),
+    url: text('url').notNull(),
+    proxyUrl: text('proxy_url').notNull(),
+    height: integer('height'),
+    width: integer('width'),
```

Comment:
I believe we could remove these extraneous fields.
```suggestion
    contentType: text('content_type'),
    url: text('url').notNull(),
    proxyUrl: text('proxy_url').notNull(),
```

Reply:
I unfortunately need `contentType` for the attachment field rendering. I can do without the rest, though.

Reply:
Done!